spec/requests/secure_cookies_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'rails_helper'
+
+RSpec.describe 'secure cookies' do
+  it 'flags all cookies sent by the application as secure, httponly, and lax' do
+    get root_url
+
+    expect(response.headers['Set-Cookie']).to include('; Secure')
+    expect(response.headers['Set-Cookie']).to include('; HttpOnly')
+    expect(response.headers['Set-Cookie']).to include('; SameSite=Lax')
+  end
+end
```

Comment:
Thank you for making a spec for this!

WDYT of also adding a unit test style test where we test different permutations of responses having multiple cookies, and or having some of the secure/httponly/samesite values set but not all? We have some other middleware tests like for `utf8_sanitizer_spec.rb`